certora/specs/CollectibleV1.spec

```diff
@@ -1,10 +1,9 @@
 methods {
-  balanceOf(address) returns (uint) envfree;
-  ownerOf(uint256 tokenId) returns (address) envfree;
-  transferable() returns (bool) envfree;
-  totalSupply() returns (uint) envfree;
-  owner() returns (address) envfree;
-  maxSupply() returns (uint) envfree;
+  function balanceOf(address) external returns (uint) envfree;
+  function ownerOf(uint256 tokenId) external returns (address) envfree;
+  function transferable() external returns (bool) envfree;
+  function totalSupply() external returns (uint) envfree;
+  function maxSupply() external returns (uint) envfree;
```

Comment:
I needed to remove `owner()` here because it seems we don't actually have such function on `CollectibleV1`